src/k8s/cmd/k8s/k8s_x_capi.go

```diff
@@ -34,6 +36,26 @@ func newXCAPICmd(env cmdutil.ExecutionEnvironment) *cobra.Command {
 			}
 		},
 	}
+	setNodeToken := &cobra.Command{
+		Use:   "set-node-token <token>",
+		Short: "Set the node token to authenticate with per-node k8sd endpoints",
+		Args:  cmdutil.ExactArgs(env, 1),
+		Run: func(cmd *cobra.Command, args []string) {
+			token := args[0]
+			if token == "" {
```

Comment:
I don't think that can happen if you set `cmdutil.ExactArgs`, can it?

Reply:
It checks for the argument count but not what the argument is so `k8s x-capi set-node-token ""` can create an empty file.

Reply:
Good to know, I though this would also be prevented so that "" would not count as an argument. TIL